### ISSUE TITLE
Enable converting for more project types

### ISSUE THREAD

**Issue by SchreinerK** [brianlagunas/ConvertProjectToCore3#7]
If I add the guids to the project manually, I could also convert other project types.
<ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
convert works also with 
- Class Library (.NET Framework)
- Unit Test Project (.NET Framework)
- Console App  (.NET Framework)
You could activate the menu item and the manual workaround would no longer be necessary.

**Comment by brianlagunas** [comment]
Yeah, I haven't really played with this in awhile. Damn those day jobs 😄 . Thanks for the tip

**Comment by AbdalaMask** [comment]
how convert to WinForms

**Comment by AbdalaMask** [comment]
This extension converts a WinForms application from the .NET Framework to .NET Core 3.
